src/client-link/entrypoint.sh

```diff
@@ -30,8 +32,8 @@ $EXPOSE {
 END
 )
 
-    else
-        export EXPOSE=$(cat <<-END
+        else
+            export EXPOSE=$(cat <<-END
 $EXPOSE {
          transport http {
             tls
```

Comment:
Think we want this to be conditional?

Reply:
Yes, if CADDY_TLS_PROXY Proxy is active, this must be added to the Caddy Config. If CADDY_TLS_INSECURE is also active, the tls_insecure_skip_verify flag must also be used.

I would try to implement a solution here tomorrow (GMT +1).

Should these parameters already be available in the `make link` mehtode so that this is also output or is documentation like FORWARD_ONLY  in the readme sufficient here?

Reply:
I think only documenting the flags in README.md is sufficient, thanks again

Reply:
I'm finished with the changes so far.

I rolled back the changes in the `create-link` section before PR #31 and only added the parameters to the `client-link`.

All 3 variants work for me.

Please review and sorry for the problems